app.rb

```diff
@@ -0,0 +1,141 @@
+require './student'
```

Comment:
- [ ] [OPTIONAL] Please don't forget to create MIT license and add it to readme 👍🏼